client/src/App.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
-import Navbar from './components/Navigation';
+import Navbar from './components/navigation';
```

Comment:
I can't compile the app. I can run the app when replacing `'./components/navigation'` with `'./components/Navigation'`

Reply:
It might be a folder naming problem.